service/dps/storm/topologies/indexing/src/test/java/eu/europeana/cloud/service/dps/storm/topologies/indexing/bolts/IndexingBoltTest.java

```diff
@@ -7,15 +7,12 @@
 import eu.europeana.cloud.common.model.Revision;
 import eu.europeana.cloud.service.dps.PluginParameterKeys;
 import eu.europeana.cloud.service.dps.storm.StormTaskTuple;
-import eu.europeana.indexing.Indexer;
-import eu.europeana.indexing.IndexerFactory;
-import eu.europeana.indexing.exception.IndexerConfigurationException;
+import eu.europeana.cloud.service.dps.storm.topologies.indexing.bolts.IndexingBolt.IndexerPoolWrapper;
+import eu.europeana.indexing.IndexerPool;
 import eu.europeana.indexing.exception.IndexingException;
-
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
-
 import org.apache.storm.task.OutputCollector;
```

Comment:
I fixed the unit tests for IndexingBoltTest, please fix the tests for IndexingTopology!